pkg/cli/options_test.go

```diff
@@ -34,6 +35,167 @@ import (
 )
 
 var _ = Describe("Discover external plugins", func() {
+	Context("with valid plugins root path", func() {
+		var (
+			homePath           string = os.Getenv("HOME")
+			customPath         string = "/tmp/myplugins"
+			xdghome            string = os.Getenv("XDG_CONFIG_HOME")
+			originalPluginPath string = os.Getenv("KB_PLUGINS_PATH")
+		)
+
+		AfterEach(func() {
+			if originalPluginPath != "" {
+				// restore the original value
+				err := os.Setenv("KB_PLUGINS_PATH", originalPluginPath)
+				Expect(err).To(BeNil())
+			} else {
+				// unset if it was originally unset
+				err := os.Unsetenv("KB_PLUGINS_PATH")
+				Expect(err).To(BeNil())
+			}
+		})
+
+		When("XDG_CONFIG_HOME is not set and using the $HOME environment variable", func() {
+			BeforeEach(func() {
+				err := os.Unsetenv("KB_PLUGINS_PATH")
+				Expect(err).To(BeNil())
+
+				err = os.Unsetenv("XDG_CONFIG_HOME")
+				Expect(err).To(BeNil())
+
+				xdghome = os.Getenv("XDG_CONFIG_HOME")
```

Comment:
Looks like we can probably remove this line - I don't see anywhere in this context where it is used.

Reply:
Thanks for catching!